HaplotypeMap/BuildHaplotypeMap.wdl

```diff
@@ -0,0 +1,167 @@
+# BuildHaplotypeMap
+#
+# This workflow creates a haplotype map file containing a set of low LD sites for fingerprinting
+#
+# Inputs:
+#
+# 1. Genotyped multisample GVCF
```

Comment:
does it need to be a GVCF, or will a VCF work?

Reply:
I think a VCF will work, I'm gonna try it just to check but I will change this if it does